app/views/hiring_staff/vacancies/application_details/edit.html.haml

```diff
@@ -36,4 +36,28 @@
         = f.gov_uk_date_field :expires_on,
                               legend_options: { page_heading: false, class: "govuk-label" }
 
+      %div.govuk-form-group#expiry_time{:class => ("govuk-form-group--error" if @application_details_form.errors[:expiry_time].present?)}
```

Comment:
We've got duplication between this view and the `new.html.haml` view.  Can we consolidate the expiry_time into one view?  The copy form will also need to use this field.

Reply:
Resolved this in e543ec7